### PR TITLE
Optional deploy-as-is

### PR DESCRIPTION
**Pull request by nvazquez**
Updated documentation according to changes on PR https://github.com/apache/cloudstack/pull/4901

### CODE REVIEW

File: source/adminguide/templates.rst [hunk 1]
```diff
@@ -249,7 +237,12 @@ To upload a template:
       available, or All Zones to make it available throughout
       CloudStack.
 
-   -  **OS Type**: (Except for VMware). This helps CloudStack and the hypervisor perform
+   - **Read VM settings from OVA**. (VMware only) If selected, the registered template will allow users to deploy VMs as clones of the template, including all their properties, configurations, end-user license agreements, disks, os type, etc. This option allow users to register virtual appliances. See `Support for Virtual Appliances <virtual_machines.html#about-virtual-appliances>`_.
```
Comment (GabrielBrascher) [inline source/adminguide/templates.rst:240]: This option **allow** :arrow_right:  **allows**

**Review by GabrielBrascher — APPROVED** [review]
Documentation looks clear about what changed. LGTM.

I've just raised a minor suggestion.

File: source/adminguide/templates.rst (hunk 1 above)
Comment (nvazquez) [reply source/adminguide/templates.rst]: Thanks @GabrielBrascher - fixed

**Comment by shwstppr** [comment]
@blueorangutan docbuild

**Review by shwstppr — APPROVED** [review]

**Comment by rhtyd** [comment]
@nvazquez can you raise the same against 4.15 branch?

**Pull request merged**